docs/teleport/README.md

```diff
@@ -0,0 +1,99 @@
+# Project Teleport Overview (Private Preview)
+
+Instancing a custom environment within seconds is one of the many wonders of running containers. Having to wait for the image and its layers to download & decompress the first time is the current price of admission.
+
+***Project Teleport removes the cost of download and decompression by mounting pre-expanded layers from the Azure Container Registry to Teleport enabled Azure container hosts.***
+
+|Dedicated VM|Teleport w/ACR Tasks |
+|-|-|
+|![](./media/vm-aci.gif)|![](./media/tasks-aci.gif)|
+
+[![](./media/AzureFridayTeleportPreviewThumb.png)](https://channel9.msdn.com/Shows/Azure-Friday/How-to-expedite-container-startup-with-Project-Teleport-and-Azure-Container-Registry/player#time=21s)
+
+![](./media/teleport-metrics.png)
+> For more background, please see [Azure Container Registry Adds Teleportation][teleport-blog-post]
+
+## Table of Contents
+
+- [Sign Up for the Project Teleport Preview](#sign-up-for-the-project-teleport-preview)
+- [Supported Services](#supported-services)
+- [Preview Constraints](#preview-constraints)
+- [Getting Started with Teleportation](./getting-started.md)
+- [Getting Support](#getting-support)
+
+## Sign Up for the Project Teleport Preview
+
+In these early stages, we're looking for direct feedback. To request access, please sign up here: [aka.ms/teleport/signup][signup]
+
+## Supported Services
+
+- ~~Preview 1 focused on running containers within [ACR Tasks][acr-tasks].~~
+- Preview 2 focuses on running containers within [AKS][aks]. See: [Getting Started with AKS][aks-getting-started]
+
+Additional services and scenarios will come online as we incorporate more feedback.
+
+## Preview Constraints
+
+Preview 2 has the following constraints. Your feedback will help us prioritize this list.
+
+- Limited to running images with [AKS][aks-getting-started]
+- Support for [premium registries][acr-tiers]
+- Registries must exist in the following regions:
+    | Region | Code |
+    | - | - |
+    | East US | EUS |
+    | East US 2 | EUS2 |
+    | South Central US | SCUS |
+    | West US | WUS |
+    | West US 2 | WUS2 |
+    | West Europe | WEU |
+  - Additional regions, including other continents will come online as we get more feedback.
+- [Geo-replicated](https://aka.ms/acr/geo-replication) registries require *translocation*. For preview 2, only the master region of a geo-replicated registry will support teleportation.
```

Comment:
We are unable to enable teleport on geo-replication enabled registry now.

Reply:
Correct. The comment is "only the master region of a geo-replicated registry will support teleportation."

Reply:
Are you saying we don't support Teleport on the master region of a geo-replicated registry? Or, that we don't replicate to other regions?

Reply:
We don't support teleport on geo-replicated registry at all.